Task3.py

```diff
@@ -0,0 +1,19 @@
+string = input()
+
+
+def decoratorHtml(left, right):
+    def decorator(function):
+        def wrapper(n1):
+            return left + function(n1) + right
+
+        return wrapper
+
+    return decorator
+
+
+@decoratorHtml("<html>", "</html>")
+def decorate(n):
+    return n
+
+
+print(decorate(string))
```

Comment:
https://docs.python.org/3/library/__main__.html

Reply:
okay, I'll fix it